FundacionParaguaya/MentorApp
# Feature/818 select performance

I've change the regular map funcion in Select component with a flat list. I've also refactored it, not performance wise, but to simplify the code. There are moderate performance boosts to the Families and Primary Participant pages.

**To Test**
1. Login to testing demo/demo. Open the Geco survey and try to complete and sync it. Check if you can use the select normally.
2. Log into d/demo and check if the demo fupa survey is displaying radio buttons correctly. You will need to go into `redux/actions` and change the load surveys action so there are no `conditionGroups`.

Get back to me when you are done.
